vpr/src/pack/pb_type_graph.cpp

```diff
@@ -1914,3 +1929,22 @@ const t_pb_graph_edge* get_edge_between_pins(const t_pb_graph_pin* driver_pin, c
 
     return nullptr;
 }
+
+/* Date:June 8th, 2024
+ * Author: Kate Thurmer
+ * Purpose: This subroutine computes the index of a pb graph node at its
+            level of the pb hierarchy; it is computed by the parent and
+            passed to each child of each child pb type. When the child is
+            a primitive, the computed indes is its flat site index.
```

Comment:
indes -> index

Reply:
Slightly better to put this comment above the function declaration instead of the implementation.